pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1196,4 +1196,7 @@ self: super: {
   beam-migrate = appendPatch super.beam-migrate ./patches/beam-migrate-fix-ghc-8.6.x-build.patch;
   beam-postgres = appendPatch super.beam-postgres ./patches/beam-postgres-fix-ghc-8.6.x-build.patch;
   beam-sqlite = appendPatch super.beam-sqlite ./patches/beam-sqlite-fix-ghc-8.6.x-build.patch;
+
+  # https://github.com/sighingnow/computations/pull/1
+  primesieve = appendPatch super.primesieve ./patches/primesieve-fix-external-libraries.patch;
```

Comment:
That patch appears to be missing.

Reply:
Please download that patch from the github pull request rather than checking it into our repository directly.

Reply:
Patch downloaded from upstream pull request.